azure-pipelines.yml

```diff
@@ -12,13 +12,43 @@ variables:
   _TeamName: DotNetSpark
   MSBUILDSINGLELOADCONTEXT: 1
   ArtifactPath: '$(Build.ArtifactStagingDirectory)\Microsoft.Spark.Binaries'
-  CurrentDotnetWorkerDir: '$(ArtifactPath)\Microsoft.Spark.Worker\netcoreapp3.1\win-x64'
+
+  backwardCompatibleRelease: '1.0.0'
+  forwardCompatibleRelease: '1.0.0'
+
+  backwardCompatibleTestOptions_Windows_2_3: ""
+  forwardCompatibleTestOptions_Windows_2_3: ""
+  backwardCompatibleTestOptions_Linux_2_3: ""
+  forwardCompatibleTestOptions_Linux_2_3: ""
+
+  backwardCompatibleTestOptions_Windows_2_4: ""
+  forwardCompatibleTestOptions_Windows_2_4: ""
+  backwardCompatibleTestOptions_Linux_2_4: ""
+  forwardCompatibleTestOptions_Linux_2_4: "--filter \
+  (FullyQualifiedName!=Microsoft.Spark.Extensions.Hyperspace.E2ETest.HyperspaceTests.TestExplainAPI)&\
+  (FullyQualifiedName!=Microsoft.Spark.Extensions.Hyperspace.E2ETest.HyperspaceTests.TestIndexCreateAndDelete)&\
+  (FullyQualifiedName!=Microsoft.Spark.Extensions.Hyperspace.E2ETest.HyperspaceTests.TestSignatures)"
```

Comment:
Do you know why these are filtered? I see no changes from 1.0.0?

Reply:
This is filtered due to the issue you found in this PR.

1.0.0 codebase is using `.Csv("Resources\\people.csv");`

Which doesn't work on linux which requires `.Csv($"{TestEnvironment.ResourceDirectory}people.csv");` to be OS agnostic.

Reply:
Can you add a comment that this is due to the test issue, not the functionality changes? And you can add a link to this PR for the fix.